### DISCUSSION TITLE
Projects: Link back to Repository

### DISCUSSION THREAD

**Discussion by Plasma** [community/community#28859] [Projects and Issues]
Great work on the new Projects GA. I'm starting to use it, but often find when I click on a Project, I wish I could quickly get back to the Repository I came from (where the Code, Issues, PRs, etc) are.

Perhaps at the top here could be a set of links to repositories (automatic from the issues, or perhaps pre-set from Project Settings?):

![image](https://user-images.githubusercontent.com/229209/183595184-b9bec797-9f46-4782-adcd-e8668107b770.png)

**Comment by dogweather** [comment]
There's kind of a complex taxonomy here: a project seems to belong to an account, plus optionally, one or more repo's. (?) Personally, I'm not sure how this should be represented.

**Comment by rileybroughten** [comment]
Thank you for your feedback! We are looking for ways to make navigation easier and more intuitive and will take this back to the team 👍
